Jenkinsfile.deploy

```diff
@@ -178,8 +179,23 @@ pipeline {
             post {
                 always {
                     saveReports(params.SKIP_TESTS)
-                    // TODO: Enable together with the profile "integration-tests" once dockerhub pull limit is resolved.
-                    // archiveArtifacts(allowEmptyArchive: true, artifacts: "**/cypress/screenshots/**,**/cypress/videos/**", fingerprint: false)
+                }
+            }
+        }
+
+        stage("Run integration tests Vehicle Routing") {
```

Comment:
Running the integration tests in a separate stage means also running the maven build twice, which prolongs the execution time of the pipeline. I would prefer if we could decide whether or not to run the IT in the original `Build ...` stage.

Reply:
Fixed c1b6a5f261ff0b981acce235924b04f0edc575e7